templates/cert-watcher

```diff
@@ -2,6 +2,20 @@
 
 # This script watches for new SSL certificates and installs them to haproxy.
 
+install_cert() {
+    # Certbot behaves differently when renewing a certificate as opposed to
+    # obtaining one for the first time.  The only way I've found to make sure
+    # all files are fully written is to wait some time before actually
+    # installing the certificate.
+    sleep 10
+    cd "{{ LETSENCRYPT_LIVE_DIR }}/$1"
+    # Haproxy expects the certificate and the private key concatenated
+    # together to a single .pem file.  We lock the certs directory to protect
+    # against haproxy reading a half-written certificate.
+    flock {{ LOAD_BALANCER_CERTS_DIR }} cat fullchain.pem privkey.pem \
```

Comment:
Since you have to flock the dir before you can write to it anyway, doesn't this basically make this operation serial, not parallel?